packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -500,29 +526,38 @@ export interface ClusterOptions extends CommonClusterOptions {
   readonly kubectlEnvironment?: { [key: string]: string };
 
   /**
-   * An AWS Lambda Layer which includes `kubectl`, Helm and the AWS CLI.
+   * An AWS Lambda Layer which includes `kubectl` and Helm.
    *
-   * By default, the provider will use the layer included in the
-   * "aws-lambda-layer-kubectl" SAR application which is available in all
-   * commercial regions.
+   * This layer is used by the kubectl handler to apply manifests and install
+   * helm charts. You must pick an appropriate releases of one of the
+   * `@aws-cdk/asset-kubectl-vXX` packages, that works with the version of
```

Comment:
Should be `@aws-cdk/layer-kubectl-vXX` packages. But I agree with @iliapolo, let's omit this until those packages are ready to go.

Reply:
Let me clarify:

When this gets released (next week's Tuesday/Wednesday) or so, the `@aws-cdk/layer-kubectl-vXX` packages will probably be done and released. So users will not see this before they are ready.

Even if the worst happens and we don't make that... it should be fairly obvious that if these packages are mentioned but they don't exist, you should wait for them to appear. If and when they do, a smidgeon of docs will already be present.

I guess I don't see a lot of detriment in advertising these packages.